src/msw/bitmap.cpp

```diff
@@ -1259,6 +1259,56 @@ void wxBitmap::MSWUpdateAlpha()
 #endif // wxUSE_WXDIB/!wxUSE_WXDIB
 }
 
+void wxBitmap::BlendMaskWithAlpha()
+{
+#if defined(wxHAS_RAW_BITMAP)
+    if ( !HasAlpha() || !GetMask() )
+        return;
+    // Update alpha channel by blending original alpha values with mask and then remove the mask.
+    // For non-masked pixels alpha channel values will remain intact and for masked pixels
+    // they will be set to the transparent value (0).
+    AllocExclusive();
+
+    {
+        wxBitmap bmpMask = GetMask()->GetBitmap();
+        wxNativePixelData maskData(bmpMask);
+        if ( !maskData )
+            return;
```

Comment:
In the wxNativePixelData ctor there is a call to wxBitmap::GetRawData() and theoretically something wrong can happen there (especially in conversion form DDB to DIB in case of mask). But wxCHECK would be a better option here.